cloudant-client/src/test/java/com/cloudant/tests/SessionInterceptorExpiryTests.java

```diff
@@ -108,9 +112,17 @@ private void queueResponses(Long expiry, String cookieValue) {
     private void executeTest(Long expiryTime, String cookieValue) throws Exception {
         queueResponses(expiryTime, cookieValue);
         HttpConnection conn = Http.GET(mockWebServer.url("/").url());
+        conn.connectionFactory = (okUsable) ? new OkHttpClientHttpUrlConnectionFactory() :
```

Comment:
Do we have a matrix of these tests which run with OK and without? I haven't really thought about that until now.

Reply:
Basically anything that extends `HttpFactoryParameterizedTest`. They work "automatically" if using the `ClientBuilder`. These tests that I added recently work directly with the `HttpConnection` though so needed this `connectionFactory` set to actually use both connection types, something that slipped my mind when I originally added the tests.